templates/openstack/consulting.html

```diff
@@ -423,7 +424,7 @@ <h3 class="p-stepped-list__title">Deploy</h3>
           <div class="row u-no-padding p-stepped-list__content">
             <div class="col-8">
               <p>
-                We deploy the cloud for you on your premises or in one of our partners’ data centres. We validate it post-deployment and benchmark its performance. At the end, we provide you with the deployment guide which outlines all the steps required to re-deploy the cloud if needed.
+                We deploy the cloud for you on your premises or in one of our partners' data centres. We validate it post-deployment and benchmark its performance. At the end, we provide you with the deployment guide which outlines all the steps required to re-deploy the cloud if needed.
```

Comment:
"centres" here has been missed - it should be AE "centers" now
```suggestion
                We deploy the cloud for you on your premises or in one of our partners' data centers. We validate it post-deployment and benchmark its performance. At the end, we provide you with the deployment guide which outlines all the steps required to re-deploy the cloud if needed.
```